keywords/TestServerNetMsft.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, version_build, host, port, community_enabled=None, platform="
             "[windows:vars]",
             "ansible_user={}".format(os.environ["LITESERV_MSFT_HOST_USER"]),
             "ansible_password={}".format(os.environ["LITESERV_MSFT_HOST_PASSWORD"]),
-            "ansible_port=5985",
+            "ansible_port=5986",
```

Comment:
can you change it back to 5985 as it won't work with .Net windows physical machine?